engines/director/lingo/lingo-object.cpp

```diff
@@ -907,8 +910,21 @@ bool BitmapCastMember::setField(int field, const Datum &d) {
 		warning("STUB: BitmapCastMember::setField(): Unprocessed setting field \"%s\" of cast %d", g_lingo->field2str(field), _castId);
 		return false;
 	case kTheRegPoint:
-		warning("STUB: BitmapCastMember::setField(): Unprocessed setting field \"%s\" of cast %d", g_lingo->field2str(field), _castId);
-		return false;
+		if (d.type == POINT || (d.type == ARRAY && d.u.farr->arr.size() >= 2)) {
```

Comment:
And what about RECTs? Those could potentially work as well, being arrays of size 4